packages/p2p/src/pages/my-profile/block-user/block-user-empty/block-user-empty.tsx

```diff
@@ -24,13 +20,7 @@ const BlockUserEmpty = () => {
                 </div>
             </DesktopWrapper>
             <MobileWrapper>
-                <MobileFullPageModal
```

Comment:
Deleted this part, since its redundant and this component and its implementation already been used by the parent component. 
--------------------
`Before changes`
--------------------
![Screenshot 2023-05-31 at 4 43 47 PM](https://github.com/farrah-deriv/deriv-app/assets/125247833/6a6de9f9-f74f-4d54-b276-853a5f3e5d4e)
--------------------
`After changes`
--------------------
![Screenshot 2023-05-31 at 4 49 55 PM](https://github.com/farrah-deriv/deriv-app/assets/125247833/e6303ef0-1967-4340-b33e-f4628c83edca)